hnn_core/extracellular.py

```diff
@@ -24,10 +24,27 @@
 from copy import deepcopy
 from numpy.linalg import norm
 from neuron import h
+import quantities as pq
 
 from .externals.mne import _validate_type, _check_option
 
 
+def csd1D_2ndD(data, ch_axis=0, delta=1):
```

Comment:
you should follow https://peps.python.org/pep-0257/ and https://peps.python.org/pep-0008/

use docstrings accordingly.